src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -118,6 +119,18 @@ case class IndexLogEntry(
 
   def relations: Seq[Relation] = source.plan.properties.relations
 
+  def allSourceFiles: Set[FileInfo] = {
```

Comment:
Question @sezruby , do you rely on full file paths in this allSourceFiles object?
Asking because with this tree structure pr #132 it FileInfo will contain only the final file name.